exp-player/addon/services/get-current-user.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Service.extend({
+    store: Ember.inject.service(),
+    load: function() {
+        // TODO work for real
+        return this.get('store'). findRecord('account', 'experimenter.accounts.tester0').then((account) => {
```

Comment:
The account model provides a helper function called `profileById` which may be useful for implementing this for real. https://github.com/abought/exp-addons/blob/ce9c7a094082ac82f9d35932eca71adb99231167/exp-models/addon/models/account.js#L18-L18

Separately, if we are not planning to implement this immediately (based on reviewing this for merge before it is there), a JIRA ticket might help to remind us and track this needed element of work.